extruct/rdfa.py

```diff
@@ -43,7 +61,28 @@ def extract_items(self, document, base_url=None, expanded=True):
                           vocab_cache_report=False,
                           refresh_vocab_cache=False,
                           check_lite=False)
-
+        document = self.expandedOGSupport(document)
         g = PyRdfa(options, base=base_url).graph_from_DOM(document, graph=Graph(), pgraph=Graph())
         jsonld_string = g.serialize(format='json-ld', auto_compact=not expanded).decode('utf-8')
         return json.loads(jsonld_string)
+
+    def expandedOGSupport(self,document):
+      prefixDic = {}
+      for head in document.xpath('//head'):
+        for el in head.xpath('meta[@property and @content]'):
```

Comment:
@ShivinDass could you please use 4 spaces for indentation, like in the rest of the project?